pkg/imager/iso/uefi.go

```diff
@@ -149,6 +152,10 @@ func CreateUEFI(printf func(string, ...any), options UEFIOptions) error {
 		return err
 	}
 
+	if _, err := cmd.Run("mcopy", "-i", efiBootImg, options.UKISigningCertDerPath, "::loader/keys/uki-signing-cert.der"); err != nil {
```

Comment:
my only concern is - should we move it to `::EFI/` so that we don't create something which would confuse systemd-boot

Reply:
yeh, that works, so `::EFI/enrollment/uki-signing-key.der` or `::EFI/keys/uki-signing-key.der`?

Reply:
lgtm!